filebeat/module/pensando/dfw/ingest/pipeline.yml

```diff
@@ -0,0 +1,215 @@
+---
+description: Pipeline for parsing Penando DFW logs 
+processors: 
+- rename:
+    field: message
+    target_field: event.original
+- grok:
+    field: event.original
+    patterns: 
+      - "%{SYSLOG5424PRI}%{NONNEGINT:syslog5424_ver} +(?:%{TIMESTAMP_ISO8601:syslog5424_ts}|-) +(?:%{IPORHOST:syslog5424_host}|-) +(-|%{SYSLOG5424PRINTASCII:syslog5424_app}) +(-|%{SYSLOG5424PRINTASCII:syslog5424_proc}) +(?::-|%{SYSLOG5424PRINTASCII:syslog5424_msgid}) +\\[%{GREEDYDATA:payload_raw}\\]$"
+- json:
+    field: payload_raw 
+    target_field: json
+- remove: 
+    field: syslog5424_sd
```

Comment:
If you want, `remove` accepts a list of fields as a `field` value if you want to combine them in a single one.

Reply:
Made this change.  Thanks, it will make it more readable.